src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery lottery = new Lottery();
+            System.out.println(lottery.getRandomBall());
+            System.out.println("");
```

Comment:
A separate `println("")` call is redundant - if you want to add an extra new line, you can use dedicated character: `"\n"` and concatenate it with the first `println` call, for example:
```java
System.out.println(lottery.getRandomBall() + "\n");
```